HEAD is now at ba7b6fe generalize
diff -cr -N -x .git -x '*.patch' /old//conf/gitolite.conf /new//conf/gitolite.conf
*** /old//conf/gitolite.conf	2016-12-23 10:57:00.243364028 -0500
--- /new//conf/gitolite.conf	2016-12-23 10:56:21.707364718 -0500
***************
*** 1,5 ****
  repo <<< guild_name >>>
-     RW+     =   <<< guild_name >>>
  
- repo testing
-     RW+     =   @all
--- 1,6 ----
+ repo @all
+     RW   refs/.*  =   <<< guild_name >>>
+     R      =   @all
+ 
  repo <<< guild_name >>>
  
